app/demo/components/date_fields.pom

```diff
@@ -96,13 +96,38 @@ Coprl::Presenters.define(:date_fields) do
     grid do
       column 6 do
         title 'As a simple text field'
-        text 'You can use the date field without a picker and get a simple auto-formatting text input.'
-        date_field picker: false do
-          label 'Birthdate'
-        end
+        text 'You can use the date field without a picker for locale-aware text input of a date.'
+        text 'When not specified, locale is obtained from the browser. It can be overridden via the `locale` attribute.'
+        attach :text_date_field
       end
     end
 
   end
   attach :code, file: __FILE__
 end
+
+Coprl::Presenters.define(:text_date_field) do
+  content id: :text_date_field do
+    select name: :locale do
+      label 'Locale'
+
+      option '', value: nil
+      option 'en-US', value: 'en-US', selected: context[:locale] == 'en-US'
+      option 'en-GB', value: 'en-GB', selected: context[:locale] == 'en-GB'
+      option 'ar-DJ', value: 'ar-DJ', selected: context[:locale] == 'ar-DJ'
+      option 'zh-TW', value: 'zh-TW', selected: context[:locale] == 'zh-TW'
+
+      event :change do
+        replaces :text_date_field, :text_date_field
+      end
+    end
```

Comment:
This looks like it's offering the user a select box of locales, is that right? Perhaps a screenshot would help me understand what's going on here.

Reply:
yes. it's for the demo web app that's embedded in the repo.

![Screenshot 2024-04-03 at 08 22 48](https://github.com/coprl/coprl/assets/29804640/d8aca4d6-24db-4e4d-b018-dd0826c249ed)